### PR TITLE
Added map function

### PR DESCRIPTION
**Pull request by Tadas-M**

### CODE REVIEW

File: University-advisor-web/Controllers/MapController.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+    public class MapController : Controller
+    {
+        // GET: /<controller>/
+        [HttpGet]
+        public IActionResult Index()
+        {
+            var model = new MapModel();
+            model.Address = "Vilnius";
+            model.Range = 500;
```
Comment (avizen-j) [inline University-advisor-web/Controllers/MapController.cs:20]: As I understand these values are default and shown to user when he opens the page?

File: University-advisor-web/Models/MapModel.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                    var newSchoolInfo = new MarkerModel
+                    {
+                        Latitude = Convert.ToDouble(lat),
+                        Longitude = Convert.ToDouble(lon),
+                        Name = name
+                    };
+                    schoolsInRange.Add(newSchoolInfo);
+                }
+            }
+            return (schoolsInRange);
```
Comment (avizen-j) [inline University-advisor-web/Models/MapModel.cs]: Why it is in brackets? :D

File: University-advisor-web/Controllers/MapController.cs (hunk 1 above)
Comment (Tadas-M) [reply University-advisor-web/Controllers/MapController.cs:20]: Yes, I think in the future we should let user to input his home address and use it as the default value. Hardcoded value is temporary solution

File: University-advisor-web/Models/MapModel.cs (hunk 2 above)
Comment (Tadas-M) [reply University-advisor-web/Models/MapModel.cs]: I will fix it right away:D

File: University-advisor-web/Tools/GeoLocationApi.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            catch (WebException ex)
+            {
+                var errorResponse = ex.Response;
+                using (var responseStream = errorResponse.GetResponseStream())
+                {
+                    var reader = new StreamReader(responseStream, Encoding.GetEncoding("utf-8"));
+                    var errorText = reader.ReadToEnd();
+                    //Add logger
+                }
+                throw;
```
Comment (avizen-j) [inline University-advisor-web/Tools/GeoLocationApi.cs]: Is necessary to rethrow exception?

File: University-advisor-web/Tools/GeoLocationApi.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                }
+            }
+            catch (WebException ex)
+            {
+                var errorResponse = ex.Response;
+                using (var responseStream = errorResponse.GetResponseStream())
+                {
+                    var reader = new StreamReader(responseStream, Encoding.GetEncoding("utf-8"));
+                    var errorText = reader.ReadToEnd();
+                    //Add logger
```
Comment (avizen-j) [inline University-advisor-web/Tools/GeoLocationApi.cs:38]: You can inject Logger through constructor and then log your error.

File: University-advisor-web/Views/Map/Index.cshtml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+            @foreach (var university in schoolsInRange)
+            {
+                @: marker = L.marker([@university.Latitude, @university.Longitude]);
+                @: marker.bindPopup("@university.Name");
+                @:universitiesInfo.push(marker);
+                @:marker.addTo(map);
+            }
+            console.log('@Model.Address')
+            console.log(@Model.Range)
```
Comment (avizen-j) [inline University-advisor-web/Views/Map/Index.cshtml]: Why do you log these values in console?

**Review by avizen-j — CHANGES_REQUESTED** [review]
Some minor fixes needs to be done.

File: University-advisor-web/Tools/GeoLocationApi.cs (hunk 4 above)
Comment (Tadas-M) [reply University-advisor-web/Tools/GeoLocationApi.cs:38]: Will do

File: University-advisor-web/Tools/GeoLocationApi.cs (hunk 3 above)
Comment (Tadas-M) [reply University-advisor-web/Tools/GeoLocationApi.cs]: Not really, ill add logger and remove throw()

**Review by avizen-j — APPROVED** [review]
Approved

**Pull request merged**